.github/workflows/frontend.yaml

```diff
@@ -0,0 +1,47 @@
+# DO NOT EDIT THIS FILE!
+#
+# It's auto-generated by sonata-project/dev-kit package.
+
+name: Frontend
+
+on:
+    schedule:
+        - cron: '30 0 * * *'
+    push:
+        branches:
+            - 4.x
+            - 5.x
+    pull_request:
+
+jobs:
+    webpack-encore:
+        name: Webpack Encore
+
+        runs-on: ubuntu-latest
+
+        steps:
+            - name: Checkout
+              uses: actions/checkout@v3
+
+            - name: Install Node
+              uses: actions/setup-node@v3
+              with:
+                  node-version: '16'
+
+            - name: Install NPM dependencies
+              uses: bahmutov/npm-install@v1
+
+            - name: Run Eslint
+              run: npx eslint assets/js
+
+            - name: Run Stylelint
+              run: npx stylelint assets/scss
+
+            - name: Run Prettier
+              run: npx prettier --check assets
+
+            - name: Run Webpack Encore
+              run: npx encore production
+
+            - name: Check if the compilation is up to date
+              run: git diff --no-patch --exit-code src/Bridge/Symfony/Resources/public
```

Comment:
This needs to be parametrized on dev-kit, by default it is src/Resources/public, but here the bundle is on src/Bridge/Symfony...

Reply:
Maybe `src/**/public` could work ?

Reply:
That could be a good idea indeed, let me try

Reply:
Fixed.